L1Trigger/L1TMuonEndCap/src/PrimitiveConversion.cc

```diff
@@ -605,20 +651,22 @@ void PrimitiveConversion::convert_gem(
     const TriggerPrimitive& muon_primitive,
     EMTFHit& conv_hit
 ) const {
-  const GEMDetId& tp_detId = muon_primitive.detId<GEMDetId>();
-  const GEMData&  tp_data  = muon_primitive.getGEMData();
+  const EMTFGEMDetId& tp_detId = emtf::construct_EMTFGEMDetId(muon_primitive);
```

Comment:
Why do we need a special GEMDetId for EMTF?

Reply:
It's just a class for convenience. The class is really basic and just holds the GEMDetId and ME0DetId, and keeps track of which is which.

Reply:
OK, got it